src/components/TaskList/TaskList.jsx

```diff
@@ -25,6 +29,11 @@ function TaskList({ isTaskCreating, onTaskCreation }) {
         return <TaskCard key={task.id} task={task} />;
     });
 
+    useEffect(() => {
+        reset();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
as reset isn't in the dependency, it's showing warning. But why we are reseting in this way? Can you think of any other better ideas?